cherryontech/ywgc
# fix: Reapply Services PR (#100)

Reapplies the squash commit from @jaynaxissa 's PR #100 .

## Context

While fixing conflicts from the #94 merge, we accidentally left out @jaynaxissa 's work from #100.

## Work done
- I used git cherry-pick with the commit sha to reapply the work done in Jayna's squash commit.
- I then manually resolved a merge conflict on the homepage. While resolving, I served the app locally and looked at the output to ensure it was as close to the designs as possible (without writing any code).
- As a bonus :P I removed a very old CSS class that was giving us a little too much margin around the sides of the app.

## Manual testing instructions
1. Serve the app locally
2. Check the app in the browser, resizing so you can see the design at all sizes.
- Expect that Nurka's content and title are showing
- Ensure that Jayna's services and background image are showing

## Gotchas/What I learned
I learned you can cherry pick a commit into the same branch.
